lightning/src/chain/chainmonitor.rs

```diff
@@ -321,6 +319,7 @@ where C::Target: chain::Filter,
 				if let Err(ref e) = persist_res {
 					log_error!(self.logger, "Failed to persist channel monitor update: {:?}", e);
 				}
+				log_trace!(self.logger, "Finished persisting Channel Monitor Update for channel {}", log_funding_info!(monitor));
```

Comment:
nit: kinda contradictory if we do fail to persist, to then log "finished persisting" right afterwards, similar above